src/storage/patch/SparqlUpdatePatchHandler.ts

```diff
@@ -136,41 +134,13 @@ export class SparqlUpdatePatchHandler extends ConvertingPatchHandler {
       metadata = new RepresentationMetadata(identifier, INTERNAL_QUADS);
     }
 
-    this.applyOperation(result, (patch as SparqlUpdatePatch).algebra);
+    // Run the query through Comunica
+    const sparql = await readableToString(patch.data);
+    const query = await this.engine.query(sparql, { sources: [ result ]}) as IQueryResultUpdate;
```

Comment:
I guess this is not a problem in most cases, but for better performance, we may consider thinking about making `Representation` implement the RDFJS `Store` interface, so that it can be passed to the query engine directly.